src/exports.cc

```diff
@@ -885,6 +885,13 @@ void wasi_unstable_proc_exit(Word /*exit_code*/) {
   context->error("wasi_unstable proc_exit");
 }
 
+Word wasi_unstable_sched_yield() { return 0; }
```

Comment:
See later comment re: adding a doc comment that spells out the hostcall type signature more specifically.